app/views/bikes/_bike_table.html.haml

```diff
@@ -1,16 +1,11 @@
 %section.full-screen-table
```

Comment:
I'm not sure this file is used anymore. If it is, please switch to `convertTime` 😉